systems/registry/org/pkg/server/org.go

```diff
@@ -289,6 +414,17 @@ func (o *OrgService) AddMember(ctx context.Context, req *pb.AddMemberRequest) (*
 	if err != nil {
 		return nil, grpc.SqlErrorToGrpc(err, "member")
 	}
+	emailReq := client.SendEmailReq{
+		To:      []string{"brackley@ukama.com"},
+		Subject: "Test Email",
+		Body:    "This is a test email.",
+		Values:  nil,
+	}
+
+	err = o.notification.SendEmail(emailReq)
```

Comment:
Why are we sending an Email to **Brackley**  every time we add a new user?

Reply:
It was a testing function I added earlier.Thanks for point that out